ui/admin/app/routes/application.js

```diff
@@ -21,6 +22,8 @@ export default class ApplicationRoute extends Route {
   @service intl;
   @service features;
   @service featureEdition;
+  @service indexedDb;
+  @service('browser/window') window;
```

Comment:
Is this a new service?

Reply:
No this is an old one we've been using so we don't directly access window globals. The primary reason is to help mocking out the `window` object when testing